generator/template_generator_app.py

```diff
@@ -280,22 +282,25 @@ def upload_spreadsheet():
             os.remove(temp_filename)
             return response
 
-# convenience method to generate a spreadsheet from a pre-yaml data structure using the schema template library's spreadsheet generator
-def _generate_spreadsheet(yaml_json):
 
+# convenience method to generate a spreadsheet from a pre-yaml data structure using the schema template library's
+# spreadsheet generator
+def _generate_spreadsheet(yaml_json):
     # I know this looks like an unused property but it somehow doesn't work if you don't declare it here first
-    # slightly complicated set-up with named temp files to pass structures around and have a persistent file object that we can actually return to the browser
+    # slightly complicated set-up with named temp files to pass structures around and have a persistent file object
+    # that we can actually return to the browser
     temp_yaml_filename = ""
     with tempfile.NamedTemporaryFile('w', delete=False) as yaml_file:
         yaml.dump(yaml_json, yaml_file)
         temp_yaml_filename = yaml_file.name
 
     with tempfile.NamedTemporaryFile('w+b', delete=False) as ssheet_file:
         temp_filename = ssheet_file.name
+        print(CONFIG_FILE)
         spreadsheet_builder = VanillaSpreadsheetBuilder(temp_filename, True)
         # TO DO currently automatically building WITH schemas tab - this should be customisable
-        spreadsheet_builder.generate_spreadsheet(tabs_template=temp_yaml_filename,
-                                              schema_urls=SCHEMA_TEMPLATE.get_latest_submittable_schema_urls(_get_ingest_api_url()), include_schemas_tab=True)
+        spreadsheet_builder.generate_spreadsheet(schema_urls=SCHEMA_TEMPLATE.get_latest_submittable_schema_urls(
```

Comment:
This literally removes the purpose of having the spreadsheet generator UI. Wranglers pick the schemas and properties they need to be in the spreadsheet, or they upload a YAML file they created and edited earlier. The generator converts the picks from the UI into a YAML-like format or reads in the uploaded file. Without the YAML config being passed to the builder, the builder can't build the spreadsheet the wrangler asked for but would just spit out a full spreadsheet with all schemas and all properties.